images/airflow/2.9.2/python/mwaa/entrypoint.py

```diff
@@ -610,6 +615,17 @@ def run_airflow_command(cmd: str, environ: Dict[str, str]):
                 ]
             )
 
+        # Hybrid runs the scheduler and celery worker processes is a single container.
+        case "hybrid":
```

Comment:
I don't particularly like the name "hybrid". Hybrid of what? In our offline discussions, we pretty much all understood what the term hybrid mean, but from the perspective of this code, it doesn't mean much. For example, if I run the following command:

```
docker run amazon-mwaa-docker-images:2.9.2 worker
```

it is clear what is being run. On other hand, no one will understand what this means:

```
docker run amazon-mwaa-docker-images:2.9.2 hybrid
```

Reply:
I do agree that the name is slightly ambiguous,  do you have a suggestion for what an alternate name should be?

The problem I see is that if we choose an explicit name like `scheduler_worker`, this is clear in isolation, but would typically become confusing in most other contexts. For example `HYBRID_SIGTERM_PATIENCE_INTERVAL` becomes  `SCHEDULER_WORKER_SIGTERM_PATIENCE_INTERVAL` which is arguably much worse than `hybrid`.

And I think `SCHEDULER_<AND/WITH>_WORKER` also has the same issue. So I am struggling to find a better alternative that does not have ambiguity.

Reply:
In cases where existing terms are not helping, I sometimes try to come up with a new, nice, and unused term that succinctly describe the new thing. Some examples I can think of:

- `mini_airflow`
- `full_airflow`
- `airflow_pod`

Probably none of these is the best (re 1, it is not necessarily mini, we could have huge compute, re 2 it is not full, it is missing web server, and so on), but I am just trying to give examples here.

In any case, it is not the end of the world if we don't come up with a better name. We can always change later, so I don't want to block you here. So, feel free to resolve the conversation.

Reply:
Yeah, it is something that if we find a better name we can always change later. The problem I see with all our names so far is they still require additional context to understand. For example, both `hybrid` and `airflow_pod` are ambiguous and would require reader having some additional background in what those terms mean in the context of our image.